tests/lib.rs

```diff
@@ -679,24 +669,14 @@ fn test_case(#[case] test_file: &str) {
     if test_data["TESTCTX"].get("CERTIFICATE").is_some() {
         let mut trustlist = TrustList::default();
 
-        let add_key_result = trustlist
-            .add_key_from_certificate(test_data["TESTCTX"]["CERTIFICATE"].as_str().unwrap());
-
-        // if the key is RSA skip the test (only EC supported)
-        let testctx_description = test_data["TESTCTX"]["DESCRIPTION"].as_str().unwrap_or("");
-        // TODO: change this once RSA is supported (see #2)
-        if testctx_description.contains("RSA") {
-            assert!(matches!(
-                add_key_result,
-                Err(KeyParseError::PublicKeyParseError(_))
-            ));
-            return;
-        }
+        trustlist
+            .add_key_from_certificate(test_data["TESTCTX"]["CERTIFICATE"].as_str().unwrap())
+            .unwrap();
 
         let (_, signature_validity) = validate(raw_hcert, &trustlist).unwrap();
         let expected_verify = test_data["EXPECTEDRESULTS"]["EXPECTEDVERIFY"]
             .as_bool()
             .unwrap_or(true);
-        assert_eq!(signature_validity.is_valid(), expected_verify);
+        assert_eq!(dbg!(signature_validity).is_valid(), expected_verify);
```

Comment:
is this `dbg!` a left-over or do we need it in the test for some reason?

Reply:
Probably a leftover. It happens, because passing tests have their output captured.